### ISSUE TITLE
Html is empty

### ISSUE THREAD

**Issue by Python-Coder-TR** [samuelhwilliams/Eel#186]
I made a main.html file and app.py file. Chrome is opening but that window is empty. My main.html is that:
```html
<html><head>
	<title>My title</title>
	<script type="text/javascript" src="/eel.js"></script>
</head>
<body>
        <h1>Hello!</h1>
</body>
</html>
```
But when I run the app.py, the html of opened window is always that:
```html
<html><head>
	<title>App</title>
	<script type="text/javascript" src="/eel.js"></script>
</head>
<body>
</body>
</html>
```
Thanks for helps.

**Comment by Python-Coder-TR** [comment]
It is very interesting that i am renamed html file as main.html to myhtml.html and edit that 
```python
 ell.start('main.html')
```
 to
```python
ell.start('myhtml.html')
```
 and it is working now. I wonder why??

**Comment by larry273** [comment]
I'm having some similar issues where eel.start("main.html") is referencing older or cached? versions of main.html and the associated js and css files and not the current files

**Comment by samuelhwilliams** [comment]
I agree it sounds like a caching issue. I have a feeling that caching should be disabled by default. I'll investigate - hopefully this weekend - and find out what's happening.

Can you share what version of Eel you're using please?

**Comment by larry273** [comment]
I managed to work around the issue by just disabling caching in chrome and refreshing the page with CTRL-F5. The eel version I'm using is 0.10.4

**Comment by ChrisKnott** [comment]
I think 1.0.0 switched to defaulting to caching off

**Comment by samuelhwilliams** [comment]
@ChrisKnott we've only released pre-release v1.0.0-a1 actually - so people installing via `pip install eel` probably won't be getting it. Might be time to bump it up to a full `v1.0.0` so that everyone gets it by default.

**Comment by samuelhwilliams** [comment]
I'm going to include the "default to caching off" in a v0.11.0 release. While this is technically a breaking change, given that we're below v1.0.0 and we think it's generally useful for most users, I think it's acceptable.

I'll close this issue off when I've released v0.11.0
